docs/platform/known-issues.md

```diff
@@ -0,0 +1,37 @@
+---
+description: known issue in the Grid Platform
+---
+
+# Known Issues
+
+## Known Issues
+
+### [BYOC](https://docs.grid.ai/platform/custom-cloud-credentials/byoc)
+Currently users on Grid Cloud and BYOC are able to create runs that resolve to the same name. This prohibits certain operations against runs/experiments. 
+
+### [Artifacts](https://docs.grid.ai/features/runs/artifacts)
+* Artifacts don't sync for fast experiments: We've detected a race condition with short-running experiments, which may cause artifacts not to be properly synced. We're working on a long-term solution for this and will fix it in the coming days. We recommend ensuring your experiments last at least a minute (to be safe), and sleep if needed as a workaround. 
+* Canceling download of Artifacts: When downloading the artifacts, Ctrl-C does not cancel the download. You can use Ctrl-Z followed  by kill -9 $(jobs -p) as a workaround.
+* Artifacts are not saved in a subdirectory: Grid artifacts are not saved in a subdirectory for run name and experiment name. This will overwrite files if two experiments have artifacts with the same file name. Workaround would be to put the artifacts in separate folders before downloading artifacts again.
+
+### [Collaboration](https://docs.grid.ai/platform/collaboration)
+Downloading artifacts and viewing logs from a teammate's experiments is not supported from the CLI. Please use the UI as a workaround.
```

Comment:
Same comment as above it applies to the rest of the document. Not an approval breaker but it is a cosmetic change we may want.